README.md

```diff
@@ -203,7 +214,7 @@ let appInsights = require("applicationinsights");
 appInsights.setup("_ikey-A_").start();
 
 // track some events manually under another ikey
-let otherClient = appInsights.createClient("_ikey-B_");

```

Comment:
I think there could've been a reason it was not exposed as a constructor. Maybe, [something along those lines](https://iron9light.wordpress.com/2008/07/24/effective-java-even-c-1-creating-and-destroying-objects/) (Item 1). What do you think?
This is public API change, so I'd like to make sure we've considered options here before changing the contract.

Reply:
The reason was that prior to my removing the main SDK class and replacing with a plain module, it wasn't actually possible to export it directly. This is already a change regardless because in the current SDK release this is called `getClient` rather than `createClient`.

I like exporting the class as it gives a direct access to what will likely become something like the base SDK in .NET, and by modifying your import statement to import just Client, you can use that base SDK directly with no other magic from the full SDK.